main.go

```diff
@@ -62,13 +64,15 @@ func main() {
 		log.Fatal().Err(err).Msg("Unable to list objects")
 	}
 
-	filtered_res := filterBucketContent(*res, opts.Suffix)
+	unproc_csvs, proc_csvs := filterBucketContent(*res, opts.Suffix, opts.ProcessedFlagSuffix)
```

Comment:
It is more common to use camelCase in go : 
```suggestion
	unprocCsvs, procCsvs := filterBucketContent(*res, opts.Suffix, opts.ProcessedFlagSuffix)
```